frontend/ui/data/onetime_migration.lua

```diff
@@ -7,7 +7,7 @@ local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
 
 -- Date at which the last migration snippet was added
-local CURRENT_MIGRATION_DATE = 20220523
+local CURRENT_MIGRATION_DATE = 20220615
```

Comment:
One last update of this date :) and we're good to go.

Reply:
That was a long birth. But it looks good ;)